### PR TITLE
chore: Add internal/focus-visible

### PR DESCRIPTION
**Pull request by gethinwebster**
*Description of changes:* Move `focus-visible` from https://github.com/cloudscape-design/components/tree/main/src/internal/hooks/focus-visible

Will follow up by removing it from there and https://github.com/cloudscape-design/chat-components/blob/main/src/internal/utils/focus-visible.ts

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by YueyingLu — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
     "react-router-dom": "^6.3.0",
+    "sass": "^1.77.6",
```
Comment (just-boris) [inline package.json:67]: Why is this here? it does not seem used
